undeploy.yaml

```diff
@@ -90,4 +90,36 @@
     - never
     - monitoring-volumes
   roles:
-    - role: undeploy/monitoring-volumes/directory
\ No newline at end of file
+    - role: undeploy/monitoring-volumes/directory
+
+# Telemetria stack -----------------------------
+
+# Delete Telemetria deployment
+- hosts: master_nodes
+  gather_facts: true
+  any_errors_fatal: true
+  tags:
+    - never
+    - telemetria
+  roles:
+    - role: undeploy/dojot-volumes/directory/telemetria
+
+# Delete telemetria pv
+- hosts: master_nodes
+  gather_facts: true
+  any_errors_fatal: true
+  tags:
+    - never
+    - telemetria-volumes
+  roles:
+    - role: undeploy/dojot-volumes/directory/telemetria
+
+# Delete monitoring nodes volume directory
+- hosts: telemetria_data_nodes
+  gather_facts: true
+  any_errors_fatal: true
+  tags:
+    - never
+    - telemetria-volumes
```

Comment:
Acredito que a telemetria faça parte dos serviços da dojot. Acho que ficaria muito complicado adicionar a tag `telemetria-volumes` aqui, pois como temos a tag `never` essa role só será executada se for passada a tag `telemetria-volumes`.

Acho que seria melhor trocar `telemetria-volumes` por `dojot-volumes`.